### PR TITLE
[enhancement](statistics) excluded delta rows num for rollup tablets

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

As title. Just record delta rows num for base tablets.

----

Co-author in #32568. Close this draft.

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Review by freemandealer — COMMENTED** [review]
The solution is LGTM, please complete this PR as well as regressions. Thanks!

File: gensrc/thrift/AgentService.thrift [hunk 1]
```diff
@@ -386,6 +386,8 @@ struct TPublishVersionRequest {
     2: required list<TPartitionVersionInfo> partition_version_infos
     // strict mode means BE will check tablet missing version
     3: optional bool strict_mode = false
+    // for delta rows statistics to exclude rollup tablets
+    4: optional set<Types.TTabletId> rollup_tablet_ids
```
Comment (freemandealer) [inline gensrc/thrift/AgentService.thrift:390]: use base_index_tablet_ids instead? to save msg space and more accurate

**Pull request closed without merging**